plugins/steelseries/fu-steelseries-fizz.h

```diff
@@ -16,3 +16,96 @@ G_DECLARE_FINAL_TYPE(FuSteelseriesFizz,
+#define STEELSERIES_FIZZ_RECEIVER_FILESYSTEM_FACTORY_SETTINGS_ID  0x03U
+#define STEELSERIES_FIZZ_RECEIVER_FILESYSTEM_MAIN_APP_ID	  0x04U
+#define STEELSERIES_FIZZ_RECEIVER_FILESYSTEM_BACKUP_APP_ID	  0x05U
+#define STEELSERIES_FIZZ_RECEIVER_FILESYSTEM_PROFILES_MOUSE_ID	  0x06U
+#define STEELSERIES_FIZZ_RECEIVER_FILESYSTEM_PROFILES_LIGHTING_ID 0x0fU
+#define STEELSERIES_FIZZ_RECEIVER_FILESYSTEM_PROFILES_DEVICE_ID	  0x10U
+#define STEELSERIES_FIZZ_RECEIVER_FILESYSTEM_PROFILES_RESERVED_ID 0x11U
+#define STEELSERIES_FIZZ_RECEIVER_FILESYSTEM_RECOVERY_ID	  0x0dU
+#define STEELSERIES_FIZZ_RECEIVER_FILESYSTEM_FREE_SPACE_ID	  0xf1U
+
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_SOFT_DEVICE_ID	0x00U
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_PROFILES_MOUSE_ID	0x06U
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_MAIN_APP_ID		0x07U
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_BACKUP_APP_ID		0x08U
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_MSB_DATA_ID		0x09U
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_FACTORY_SETTINGS_ID	0x0aU
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_FSDATA_FILE_ID	0x0bU
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_MAIN_BOOT_ID		0x0cU
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_RECOVERY_ID		0x0eU
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_PROFILES_LIGHTING_ID	0x0fU
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_PROFILES_DEVICE_ID	0x10U
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_FDS_PAGES_ID		0x12U
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_PROFILES_BLUETOOTH_ID 0x13U
+#define STEELSERIES_FIZZ_MOUSE_FILESYSTEM_FREE_SPACE_ID		0xf0U
+
+gchar *
+fu_steelseries_fizz_version(FuDevice *device, gboolean tunnel, GError **error);
+gboolean
+fu_steelseries_fizz_write_access_file(FuDevice *device,
+				      gboolean tunnel,
+				      guint8 fs,
+				      guint8 id,
+				      const guint8 *buf,
+				      gsize bufsz,
+				      FuProgress *progress,
+				      GError **error);
+gboolean
+fu_steelseries_fizz_erase_file(FuDevice *device,
+			       gboolean tunnel,
+			       guint8 fs,
+			       guint8 id,
+			       GError **error);
+gboolean
+fu_steelseries_fizz_reset(FuDevice *device, gboolean tunnel, guint8 mode, GError **error);
+gboolean
+fu_steelseries_fizz_file_crc32(FuDevice *device,
+			       gboolean tunnel,
+			       guint8 fs,
+			       guint8 id,
+			       guint32 *calculated_crc,
+			       guint32 *stored_crc,
+			       GError **error);
+gboolean
+fu_steelseries_fizz_read_access_file(FuDevice *device,
+				     gboolean tunnel,
+				     guint8 fs,
+				     guint8 id,
+				     guint8 *buf,
+				     gsize bufsz,
+				     FuProgress *progress,
+				     GError **error);
+gboolean
+fu_steelseries_fizz_write_file(FuDevice *device,
+			       gboolean tunnel,
+			       guint8 fs,
+			       guint8 id,
+			       FuFirmware *firmware,
+			       FuProgress *progress,
+			       FwupdInstallFlags flags,
+			       GError **error);
+gboolean
+fu_steelseries_fizz_battery_level(FuDevice *device, gboolean tunnel, guint8 *level, GError **error);
```

Comment:
_get_battery_level perhaps

Reply:
again, the command is named `BATTERY_LEVEL`; even if it would be `GET_BATTERY_STATUS` as the returned value as the `CHARGING_BIT` and the remaining bits are the `LEVEL`.